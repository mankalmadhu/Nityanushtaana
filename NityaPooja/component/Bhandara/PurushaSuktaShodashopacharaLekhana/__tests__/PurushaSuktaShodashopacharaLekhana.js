import React from 'react';
import ಪುರುಷಸೂಕ್ತಶೋಢಷೋಪಚಾರಲೇಖನ from '../PurushaSuktaShodashopacharaLekhana';
import Renderer from 'react-test-renderer';

jest.mock('../../../Avarana/Avarana',()=>'ಆವರಣ');
jest.mock('../../../Lekhana/Lekhana', () => 'ಲೇಖನ' );
jest.mock('../../../LekhanaSheershike/LekhanaShershike', () => 'ಲೇಖನಶೀರ್ಷಿಕೆ' );

test("ಪುರುಷಸೂಕ್ತಶೋಢಷೋಪಚಾರಲೇಖನ", () => {
  expect(Renderer.create(
    <ಪುರುಷಸೂಕ್ತಶೋಢಷೋಪಚಾರಲೇಖನ />
  ).toJSON()).toMatchSnapshot();

});