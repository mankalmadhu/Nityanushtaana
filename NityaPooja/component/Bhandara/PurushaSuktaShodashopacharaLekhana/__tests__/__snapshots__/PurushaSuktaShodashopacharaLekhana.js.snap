// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಪುರುಷಸೂಕ್ತಶೋಢಷೋಪಚಾರಲೇಖನ 1`] = `
<ಆವರಣ>
  <ಲೇಖನಶೀರ್ಷಿಕೆ>
    ಪುರುಷಸೂಕ್ತ ಶೋಢಷೋಪಚಾರ ಪೂಜಾ
  </ಲೇಖನಶೀರ್ಷಿಕೆ>
  <ಲೇಖನ />
</ಆವರಣ>
`;
